src/Selenium2Driver.php

```diff
@@ -240,27 +205,27 @@ protected static function charToOptions($char, $modifier = null)
             $ord = $char;
         }
 
-        $options = array(
-            'keyCode'  => $ord,
-            'charCode' => $ord
-        );
+        $options = [
+            'keyCode' => $ord,
+            'charCode' => $ord,
+        ];
 
         if ($modifier) {
-            $options[$modifier.'Key'] = 1;
+            $options[$modifier . 'Key'] = 1;
         }
 
         return json_encode($options);
     }
 
     /**
-     * Executes JS on a given element - pass in a js script string and {{ELEMENT}} will
```

Comment:
1. It shows up as JS errors in my IDE
2. It seems to be replaced by webdriver: https://github.com/uuf6429/MinkSelenium2Driver/blob/05872053bd733959b25fa4df3f3aa41bef845d64/src/Selenium2Driver.php#L251-L251